packages/frontend/src/components/campaigns/index.tsx

```diff
@@ -86,6 +141,16 @@ export function Campaigns() {
                     value={search}
                     onChange={handleSearchChange}
                 />
+                <Select
+                    options={statusOptions}
+                    value={status}
+                    onChange={handleStatusChange}
+                    placeholder="Status"
+                    messages={{
+                        noResults: "no",
```

Comment:
Same here.

Reply:
I just left the empty string there. The options are hardcoded so we will never see this.